cli/internal/terraform/terraform/aws/main.tf

```diff
@@ -34,6 +35,12 @@ resource "random_id" "uid" {
   byte_length = 4
 }
 
+resource "random_password" "initSecret" {
```

Comment:
Since this password is generated in Terraform it will be stored in Terraform's statefile. Is this an attack vector we might be concerned about? I think we should mention this in our docs.
See Terraform docs regarding secrets in statefile: https://developer.hashicorp.com/terraform/language/state/sensitive-data

Reply:
I think we should mention this when we finally expose Terraform to the user. Currently the statefile is managed by us and not mentioned in the docs.